### PR TITLE
[Linux] Use Steam Runtime with Proton

### PR DESCRIPTION
**Pull request by flavioislima**
Use SteamRuntime if available when using Proton.
- Check if soldier is available and use it
- if not, use scout

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: electron/gog/games.ts [hunk 1]
```diff
@@ -335,7 +335,11 @@ class GOGGame extends Game {
       let wineFlag = ['--wine', wineVersion.bin]
       let winePrefixFlag = ['--wine-prefix', winePrefix]
       if (wineVersion.type === 'proton') {
-        wineFlag = ['--no-wine', '--wrapper', `'${wineVersion.bin}' run`]
+        const runtime = getSteamRuntime()
+        const runWithRuntime = runtime.path
```
Comment (CommandMC) [inline electron/gog/games.ts:339]: Maybe a `logWarning` if a runtime isn't found would be good

**Review by CommandMC — CHANGES_REQUESTED** [review]
`runWineCommand` also needs some changes to use the runtime there as well

File: electron/gog/games.ts (hunk 1 above)
Comment (flavioislima) [reply electron/gog/games.ts]: Added the warning now

**Comment by flavioislima** [comment]
@CommandMC I will leave the `runWineCommand` to another PR for now, but added a TODO for reminder.

**Pull request merged**